packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -97,8 +154,44 @@ AppConfig.extraGetPropsArgs = (locals = {}) => {
     }
 }
 
+AppConfig.getProps = async ({api}) => {
+    const localeConfig = await getLocaleConfig({
+        getUserPreferredLocales: () => {
+            // CONFIG: This function should return an array of preferred locales. They can be
+            // derived from various sources. Below are some examples of those:
+            //
+            // - client side: window.navigator.languages
+            // - the page URL they're on (example.com/en-GB/home)
+            // - cookie (if their previous preference is saved there)
+            //
+            // If this function returns an empty array (e.g. there isn't locale in the page url),
+            // then the app would use the default locale as the fallback.
+
+            // NOTE: Your implementation may differ, this is jsut what we did.
+            //
+            // Since the CommerceAPI client already has the current `locale` set,
+            // we can use it's value to load the correct messages for the application.
+            // Take a look at the `app/components/_app-config` component on how the
+            // preferred locale was derived.
```

Comment:
Minor: to help point which function they should be looking at.
```suggestion
            // Take a look at `AppConfig.restore` on how the
            // preferred locale was derived.
```